src/bin/pgcopydb/ld_stream.c

```diff
@@ -312,7 +312,7 @@ stream_init_context(StreamContext *privateContext, StreamSpecs *specs)
 	privateContext->previous.action = STREAM_ACTION_UNKNOWN;
 
 	privateContext->lastWrite = 0;
-
+	privateContext->lastWrittenLSN = InvalidXLogRecPtr;
```

Comment:
I could have avoided `if...else` later in the PR, only if I could find a better name by which I could have assigned `lastWrittenLSN` to `startpos` and that conveys correct intention.
It still seem fine to me, but I'm open to better names even with the current intentions.